### DISCUSSION TITLE
Make the draggable widget not fully refresh when the "enable" attribute changes

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#8047] [Ideas]
I think we should make the draggable widget a bit more user friendly
It shouldn't fully refresh when the `enable` attribute changes
It should only update the `draggable` attribute and add/remove the EventListeners
In my TiddlyFlex plugin I'm doing this [here](https://burningtreec.github.io/TiddlyFlex/#%24%3A%2Fplugins%2FBTC%2FTiddlyFlex%2Fmodules%2Fwidgets%2Ftiddlyflex-draggable.js) and [here](https://burningtreec.github.io/TiddlyFlex/#%24%3A%2Fplugins%2FBTC%2FTiddlyFlex%2Fmodules%2Futils%2Fdom%2Ftiddlyflex-utils.js) and it works reliably
I need this because the draggable widget wraps the whole View- and EditTemplate inside and if it fully refreshs when the draggable attribute changes then Tiddlers in edit mode do focus the title input which causes the page to scroll chaotically
What do you think?

**Comment by Jermolene** [comment]
Hi @BurningTreeC that sounds like a worthwhile enhancement. Would you be able to work on a PR?

**Reply by BurningTreeC** [reply]
Hi @Jermolene I've created PR #8049 for this enhancement
